docs/docs/guides/operate/managing-concurrency.md

```diff
@@ -14,105 +14,97 @@ This article assumes familiarity with [assets](/guides/build/assets/) and [jobs]
 
 :::
 
-## Limit how many runs can be in progress at the same time
-
+## Limit the number of total runs that can be in progress at the same time
 
 * Dagster Core, add the following to your [dagster.yaml](/guides/deploy/dagster-yaml)
 * In Dagster+, add the following to your [deployment settings](/dagster-plus/deployment/management/deployments/deployment-settings-reference)
 
 ```yaml
-run_queue:
-  max_concurrent_runs: 15
+concurrency:
+  runs:
+    max_concurrent_runs: 15
 ```
 
+## Limit the number of runs that can be in progress for a set of ops
 
-## Limit how many runs can be in progress by tag
+You can assign assets and ops to concurrency pools which allow you to limit the number of in progress runs containing those assets or ops.  You first assign your asset or op to a concurrency pool using the `pool` keyword argument.
 
-You can configure a limit for runs that are tagged with a specific tag key or key-value pair.
+<CodeExample filePath="docs_beta_snippets/docs_beta_snippets/guides/operate/concurrency-pool-api.py" language="python" title="Specifying pools on assets and ops" />
```

Comment:
Sorry, I suggested the wrong property earlier -- this should be `path` instead of `filePath`

Reply:
🙏